src/mudlet.pro

```diff
@@ -204,6 +204,18 @@ isEmpty( OWN_QTKEYCHAIN_TEST ) | !equals( OWN_QTKEYCHAIN_TEST, "NO" ) {
   DEFINES += INCLUDE_OWN_QT5_KEYCHAIN
 }
 
+########## Full MSYS2 (alternative Windows build) environment support ##########
```

Comment:
You already have a wiki link in here for documentation, perhaps all this extra info could be put on the wiki as well? This could be a link to the wiki and a mention that it can be enabled using `WITH_MAIN_BUILD_SYSTEM="NO"`

Reply:
I don't know about other people but if I am hacking around in an unfamiliar project I want the configuration details to hand in the file I am looking through looking for clues - and not go looking for on-line reference material which has been forgotten. Fair enough that *shouldn't* happen here* but it is a lot simpler to document stuff here as I need to and then perhaps migrate it to an external reference (Wiki) when it is a bit more stable... 

... OTOH I will think about it.

Reply:
I have thought about it and my opinion is that the text present says:
* that it is only relevant for Windows developers
* that there is more information in the Wiki
* that it changes some paths and filenames
* that to enable it you need to set an environmental variable

It doesn't strike me that any of that is unhelpful to have there or that it is unduly verbose. What alternative text would you consider acceptable instead?

Reply:
The wiki link clearly states it is windows, and MSYS2. You could just as easily convey this information as 
```
# Windows MSYS2 builds require some altered paths
# see https://wiki.mudlet.org/w/Compiling_Mudlet#Compiling_on_Windows_7.2B_.28MSYS2_Alternative.29
# enable using WITH_MAIN_BUILD_SYSTEM="NO"
```

Half the lines, all the relevant information, and the link to the wiki for further information. You're not necessarily meant to be having a friendly chat over coffee with every future developer who comes across your comments